src/interfaces/IERC5564Announcer.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+/// @dev Interface for calling the `ERC5564Announcer` contract, which emits an `Announcement` event
+/// to broadcast information about a transaction involving a stealth address. See
+/// [ERC-5564](https://eips.ethereum.org/EIPS/eip-5564) to learn more.
+interface IERC5564Announcer {
+  /// @dev Emitted when something is sent to a stealth address.
+  /// @param schemeId The applied stealth address scheme (such as secp25k1).
+  /// @param stealthAddress The computed stealth address for the recipient.
+  /// @param caller The caller of the `announce` function that emitted this event.
+  /// @param ephemeralPubKey Ephemeral public key used by the sender.
```

Comment:
nit: should this clarify "used by the sender *to encrypt the Announcement metadata*"

Reply:
Updated in e204775. Note that unlike umbra we're not encrypting any data in the announcement, we just use the shared secret generated from the ephemeral pubkey to directly generate the stealth address